metaflow/plugins/argo/argo_workflows.py

```diff
@@ -2015,6 +2015,7 @@ def _container_templates(self):
                         kubernetes_sdk.V1Container(
                             name=self._sanitize(node.name),
                             command=cmds,
+                            termination_message_policy= "FallbackToLogsOnError",
```

Comment:
can you also enable it for `@kubernetes` so that there is parity?

Reply:
done.